python/lib/scheduler/dmod/scheduler/job/job_manager.py

```diff
@@ -532,31 +529,37 @@ def delete_job(self, job_id) -> bool:
         ValueError
             If no job exists with given job id.
         """
-        logging.debug("Deleting job {}".format(job_id))
+        if not isinstance(max_attempt_count, int) or max_attempt_count < 1:
```

Comment:
It's mostly a CYA habit that meshes ok with the following "is it an `int` and is it at least one?" check that happens immediately. **_IDEALLY_**, that'd come from an environment variable with a fallback since it falls in line with a system tuning setting.

I can go ahead and change it if you feel strongly about it.

Reply:
Yeah, could you please change it? It makes it a lot easier to follow.